Toolchain/default.nix

```diff
@@ -26,6 +26,8 @@ mkShell.override { stdenv = gccStdenv; } {
       texinfo
       unzip
       wget
+      # For building Jakt
+      llvmPackages_19.llvm
```

Comment:
llvmPackages_20 is stable, `llvmPackages_21` is RC.

Reply:
I used that one because it's used below to get clang.